File: backend/src/main/java/com/logistics/domain/dispatch/domain/DispatchStatus.java
package com.logistics.domain.dispatch.domain;

/** 배차완료 → 배송중 → 배송 종료. 배송 시작 전에는 취소할 수 있다. */
public enum DispatchStatus {
    REGISTERED,
    IN_TRANSIT,
    COMPLETED,
    CANCELLED,
}

File: frontend/src/constants/dispatchStatus.ts
import { DispatchStatus } from '@/types/dispatch';
import { Tone } from '@/types/ui';

export const DISPATCH_STATUS_LABEL: Record<DispatchStatus, string> = {
  REGISTERED: '배차완료',
  IN_TRANSIT: '배송중',
  COMPLETED: '배송종료',
  CANCELLED: '취소',
};

export const DISPATCH_STATUS_TONE: Record<DispatchStatus, Tone> = {
  REGISTERED: 'blue',
  IN_TRANSIT: 'green',
  COMPLETED: 'gray',
  CANCELLED: 'red',
};
